mempool/metrics.go

```diff
@@ -40,4 +40,8 @@ type Metrics struct {
 	// Number of times transactions were received more than once.
 	//metrics:Number of duplicate transaction reception.
 	AlreadyReceivedTxs metrics.Counter
+
+	// Experimental Number of connections being actively used for gossiping
+	// transactions outbound.
+	ExperimentalOutboundPeers metrics.Counter
```

Comment:
Two questions here:
* Does a metric really need to be experimental? (config knobs are a different story)
* Checking the [metric types](https://prometheus.io/docs/concepts/metric_types/), I'm not sure you want this to be a _counter_ (check the sentence saying "Do not use a counter to expose a value that can decrease"). Shouldn't it rather be a Gauge?

Reply:
I agree. I renamed it and changed the type to gauge.